code/datums/gamemodes/infestation.dm

```diff
@@ -317,6 +317,5 @@ Sensors indicate [numXenosShip || "no"] unknown lifeform signature[numXenosShip
 		var/turf/victim_turf = get_turf(victim) //Sneaky people on lockers.
 		if(QDELETED(victim_turf) || victim_turf.z != z_level)
 			continue
-		victim.adjustFireLoss(victim.maxHealth * 4)
-		victim.death()
+		victim.setFireLoss(victim.maxHealth*2)
```

Comment:
If you're going to let marines live, why not let xenos live, too? Just remove this line completely

Reply:
I only made a revert PR to see if the community prefers this. I'll update this accordingly if the PR gets an overwhelming feedback.

Reply:
Specify in the PR then that it only lets marines survive for now, until you make that change